content/21.nutraceuticals.md

```diff
@@ -178,22 +272,56 @@ However, in the case of probiotics and respiratory infections, the evidence seem
+
+### Discussion
+
+The purported prophylactic and therapeutic benefits of dietary supplements and nutraceuticals for multiple disorders, diseases, and infections has been the subject of significant research and debate for the last few decades. 
+Inevitably, scientists are also investigating the potential of these various products to treat or prevent COVID-19.
+This interest also extends to consumers, which led to a remarkable increase of sales of dietary supplements and nutraceuticals throughout the pandemic due to a desire to obtain additional protections from infection and disease.
+It is pertinent to clarify that the nutraceuticals discussed in this review, namely vitamin C, vitamin D, n-3 PUFA, zinc, and probiotics, have been selected because of their possible relevance to the biological mechanisms that can beneficially affect viral and respiratory infections and because they are currently under clinical investigation.
+However, there are various other products and molecules that have garnered scientific interest and are under further investigation. 
+These include polyphenols, lipid extracts, and tomato-based nutraceuticals among others, which have all been suggested for the prevention of potential cardiovascular complications of COVID-19 such as thrombosis [@doi:10.3389/fnut.2020.583080; @doi:10.3390/nu12061718].
+These are just some of the many substances and supplements under investigation that currently have limited evidence to support their use or supplementation with the aim to prevent or treat COVID-19.
+Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods [@doi:10.1038/s41421-020-0153-3] and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects [@doi:10.3389/fnut.2020.583080; @doi:10.3390/antiox10010047; @doi:10.3390/nu13010047].
+Notably, melatonin, vitamin D and zinc were included in the treatment plan of the Former President of the United States upon hospitalization due to COVID-19 [@doi:10.1126/science.abf0974].
+While there is plenty speculation surrounding the use of supplements, a survey of 327,720 users of the COVID Symptom Study App found that the consumption of n-3 PUFA supplements, probiotics, multivitamins, and vitamin D was associated with a lower risk of SARS-CoV-2 infection in women but not men after adjusting for potential confounders [@doi:10.1101/2020.11.27.20239087].
+According to the authors, the sexual dimorphism observed may in part be because supplements may better support females due to known differences between the male and female immune systems or it could be due to behavioural and health consciousness differences between the sexes [@doi:10.1101/2020.11.27.20239087].
```

Comment:
```suggestion
According to the authors, the sexual dimorphism observed may in part be because supplements may better support females due to known differences between the male and female immune systems, or it could be due to behavioral and health consciousness differences between the sexes [@doi:10.1101/2020.11.27.20239087].
```